### PR TITLE
Added fix for None docker_image variable when submitting a spark job

### PR DESCRIPTION
**Pull request by besil**
See https://github.com/mesosphere/dcos-spark/issues/6

### CODE REVIEW

**Comment by ssk2** [comment]
Thanks @besil! Will get this reviewed soon.

File: dcos_spark/cli.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def master():
 
 def run_spark_job(args):
     docker_image = args.get('--docker-image', constants.spark_mesos_image)
+    if docker_image is None: docker_image = constants.spark_mesos_image
```
Comment (tamarrow) [inline dcos_spark/cli.py]: Our style is to not inline `if` statements. Ie:
if x:
    y = x

**Comment by tamarrow** [comment]
Can you confirm that this change fixes the problem? It's weird that the default value doesn't work as expected.

File: dcos_spark/cli.py (hunk 1 above)
Comment (tnachen) [inline dcos_spark/cli.py]: I'll fix the style.

**Comment by tnachen** [comment]
Thanks for the fix, definitely wierd why .get with default failed to actually get the default :(

**Comment by elingg** [comment]
yeah, weird. I was able to reproduce the issue with python 2.7.5 as well

**Pull request merged**